.projenrc.ts

```diff
@@ -52,7 +52,23 @@ project.package.addField('version', version)
 project.package.addField('overrides', {'@types/babel__traverse': 'ts3.9'})
 
 // ANCHOR ESLint and prettier setup
-addLinters({project, lintPaths: ['.projenrc.ts', 'src']})
+addLinters({
+  project,
+  lintPaths: ['.projenrc.ts', 'src'],
+  extraEslintConfigs: [
+    {
+      overrides: [
```

Comment:
Why do we need this override? I tried deleting it and delete the `eslint-disable` directive from these files - it did not throw any additional warnings. Since these files have the same rule disabled (but with a slightly different directive) they seem a good candidate for removal along with all the others.

If I miss anything and it need be done this way, please add a description in the PR on why this override is needed.

Reply:
Yeah, it can be removed as well. Didn't understand at first.